.cirrus.yml

```diff
@@ -138,6 +138,13 @@ task:
       -var "task_name=${CIRRUS_TASK_NAME}" \
       -var gcp_project=$GCP_PROJECT \
       "$PACKERFILE"
+
+  set_image_script: |
+    gcloud compute images update \
+      --project "$GCP_PROJECT" \
+      --family pg-ci-${CIRRUS_TASK_NAME} \
+      pg-ci-${CIRRUS_TASK_NAME}-${DATE}
+
     gcloud compute images add-iam-policy-binding \
```

Comment:
The images update should happen *after* the permission has been added successfully. Otherwise the race that somebody might want to use the image-family, before the policy binding has been added, is back.